### PR TITLE
Kim mb duplicate ppm home path

### PR DESCRIPTION
**Pull request by kimallen**
## Description

Adds a new path to the PPM home page: `/ppm`

As an aside, I also added the number of the shipment on the shipment list on the home page since it was missing (eventually only if there is > 1 shipment, but tests get complicated, so that will have to be when we add ppm support)

## Reviewer Notes

Does this cover the story?  It felt too small to be what was expected.  
I can go to the `/ppm` page anytime after creating a profile and continue the flow from there.  I tested all the way through payment request approval.

## Setup

Create a new service member
Create a profile. 
Go to `/ppm` and see the ppm home page.
Click on the button `Continue move setup` to take you to orders.
Keep moving through the flow and going back to /ppm to see the status changes.

## Code Review Verification Steps

* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-3850) for this change

## Screenshots

image

image

### CODE REVIEW

**Review by mariaclaratgolfo — APPROVED** [review]

**Review by sojeri — COMMENTED** [review]
it looks like this branch is off another branch. can you at me or re-request my review when you've synced with latest and it's possible to review only the changes for this PR?

**Comment by kimallen** [comment]
> it looks like this branch is off another branch. can you at me or re-request my review when you've synced with latest and it's possible to review only the changes for this PR?

It should only be off of master.  I just pushed up the latest master.

**Comment by kimallen** [comment]
> > it looks like this branch is off another branch. can you at me or re-request my review when you've synced with latest and it's possible to review only the changes for this PR?
> 
> It should only be off of master. I just pushed up the latest master.

Now for real for real the latest master is pushed up.

File: src/components/Customer/Home/ShipmentList/index.jsx [hunk 1]
```diff
@@ -23,7 +23,11 @@ const ShipmentListItem = ({ shipment, onShipmentClick, shipmentNumber }) => {
       role="button"
       tabIndex="0"
     >
-      <strong>{shipment.shipmentType}</strong> <span>{shipment.id}</span> <EditIcon className={styles.edit} />
+      <strong>
+        {shipment.shipmentType}
+        {` ${shipmentNumber}`}
```
Comment (sojeri) [inline src/components/Customer/Home/ShipmentList/index.jsx:28]: interesting that we already have the number around but weren't using it! nice add-on :)

**Review by sojeri — APPROVED** [review]
looks good to me! nice to see these shipment numbers in the list :)

File: src/scenes/MyMove/index.jsx [hunk 2]
```diff
@@ -112,6 +112,7 @@ export class AppWrapper extends Component {
               {!this.state.hasError && !props.swaggerError && (
                 <Switch>
                   <Route exact path="/" component={Landing} />
+                  <Route exact path="/ppm" component={Landing} />
```
Comment (sojeri) [inline src/scenes/MyMove/index.jsx:115]: actually-- now that I already approved, I'm wondering if we should be dupeing Landing here as well to be PPMLanding or something similar so we can leave that unchanged while modifying Landing to hold the new home page?

**Pull request merged**